playbooks/vars/all.yml

```diff
@@ -4,6 +4,7 @@ gating_overrides_file: "/etc/openstack_deploy/user_zzz_gating_variables.yml"
 default_gating_overrides:
   maas_notification_plan: npTechnicalContactsEmail
   nova_virt_type: qemu
+  nova_cross_az_attach: True
```

Comment:
Looks like there was an attempt to [remove](https://github.com/rcbops/rpc-openstack/pull/2098) this override, however support wanted it left as is.  As such, I think we should remove this override from gating and exclude any failing tests from running.  Perhaps running `all` tests is not going to be possible?